SPARCore/GADArticle.m

```diff
@@ -51,57 +89,40 @@ - (void) fetchFulltextWithCompletion: (void(^_Nonnull)(GADRemoteModel * *_Nullab
     
 }
 
-+ (NSArray <GADArticle *> *) articlesFromJSON: (NSData *)json {
++ (NSArray *) fullArticleFromJsonData: (NSData *)json {
```

Comment:
I think this can be combined with `articlesFromJSON:` (accessing an undefined key on an `NSDictionary` returns `nil`, which we can legally assign to a property in Objective-C.